rules/order.js

```diff
@@ -1,9 +1,12 @@
+// Order inspired by Concentric CSS
+// http://rhodesmill.org/brandon/2011/concentric-css/
 module.exports = {
   'order/properties-order': [
     {
       order: 'strict',
       properties: [
         'content',
+        'quotes',
       ],
     },
     {
```

Comment:
Putting `z-index` between `position` and the things that matter to position (top/right/bottom/left) seems strange to me.